tests/integration/utils/test_normalizers.py

```diff
@@ -32,3 +37,33 @@ def test_normalize_journal_title(app):
     normalized_journal_title = normalize_journal_title(journal_title)
 
     assert abbreviated_journal_title == normalized_journal_title
+
+
+def test_convert_to_journal():
+    obj = MockObj()
+    formdata = {
+        'series_title': 'Physical Review',
+        'journal_title': '',
+        'type_of_doc': 'book'
+    }
+
+    expected = {'series_title': 'Physical Review',
+                'journal_title': 'Phys.Rev.',
+                'type_of_doc': 'book'}
+    result = normalizers.normalize_journal_title(obj, formdata)
+    assert expected == result
+
+
+def test_find_book():
+    obj = MockObj()
+    formdata = {
+        'book_title': 'The Large Hadron Collider',
+        'parent_book': '',
+        'type_of_doc': 'chapter'
+    }
+
+    expected = {'book_title': 'The Large Hadron Collider',
+                'parent_book': "http://localhost:5000/api/literature/1373790",
+                'type_of_doc': 'chapter'}
+    result = normalizers.find_book_id(obj, formdata)
+    assert expected == result
```

Comment:
These tests are not in the right place: they should be in `tests/integration/literaturesuggest/test_normalizers.py`. On the other hand, this points to the fact that we shouldn't have `inspirehep/utils/normalizers.py`, as the method it provides is used exactly once.